endpoints/v2/blob.py

```diff
@@ -332,6 +343,18 @@ def upload_chunk(namespace_name, repo_name, upload_uuid):
     # Upload the chunk for the blob.
     _upload_chunk(uploader)
 
+    if app.config.get("FEATURE_QUOTA_MANAGEMENT", False):
+        quota = quota_registry_model.verify_namespace_quota_during_upload(
+            namespace_name, repository_ref
+        )
+        if quota["severity_level"] == 2:
+            return Response(
+                status=413,
```

Comment:
I think either a 403 or 433 might be more appropriate here. 413 would indicate something wrong with the request itself, which is not the case.

Reply:
Any reason not using `QuotaExceeded` that you defined?